python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -130,7 +135,55 @@ def update_trip_stop_counts(db_manager: DatabaseManager) -> None:
         .values(stop_count=new_stop_counts_cte.c.stop_count)
     )
 
+    process_logger = ProcessLogger("l0_trips.update_trip_stop_counts")
+    process_logger.log_start()
+    db_manager.execute(update_query)
+    process_logger.log_complete()
+
+
+def update_static_trip_id_guess_exact(db_manager: DatabaseManager) -> None:
+    """
+    Update static_trip_id_guess with exact matches between rt and static trips
+    """
+    rt_static_match_cte = (
```

Comment:
exact trip matching is straight forward. matching on `trip_id` between RT and Static records.